plugins/auth-backend-module-okta-provider/src/module.test.ts

```diff
@@ -68,7 +65,7 @@ describe('authModuleOktaProvider', () => {
     expect(startUrl.pathname).toBe('/oauth2/v1/authorize');
     expect(Object.fromEntries(startUrl.searchParams)).toEqual({
       response_type: 'code',
-      scope: combinedScopes,
+      scope: 'openid email profile offline_access groups phone',
```

Comment:
With the latest updates this now properly merges scopes again, regarding https://github.com/backstage/backstage/pull/24743#discussion_r1598361916